modules/localization/msf/local_pyramid_map/ndt_map/ndt_map_pool.cc

```diff
@@ -0,0 +1,34 @@
+/******************************************************************************
+ * Copyright 2019 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#include "modules/localization/msf/local_map/ndt_map/ndt_map_pool.h"
+
+namespace apollo {
+namespace localization {
+namespace msf {
+
+/**@brief Constructor
+ * @param <pool_size> The memory pool size.
+ * @param <thread_size> The thread pool size.
+ */
+NdtMapNodePool::NdtMapNodePool(unsigned int pool_size, unsigned int thread_size)
+    : BaseMapNodePool(pool_size, thread_size) {}
+
+BaseMapNode* NdtMapNodePool::AllocNewMapNode() { return new NdtMapNode(); }
```

Comment:
Use safe pointers.

Reply:
we will manager the resource in MapNodePool.